pkg/kube/controllers/boshdeployment/validating_webhook.go

```diff
@@ -69,6 +76,34 @@ func NewValidator(log *zap.SugaredLogger, config *config.Config) admission.Handl
 	}
 }
 
+// OpsResourceExist verify if a resource exist in the namespace,
+// it will check it´s existance during 5 seconds,
+// otherwise it will timeout.
+func (v *Validator) OpsResourceExist(ctx context.Context, specOpsResource bdv1.Ops, ns string) (bool, string) {
+	timeOut := time.After(5 * time.Second)
```

Comment:
Could we reuse these values or parameterize them:

https://github.com/cloudfoundry-incubator/cf-operator/blob/38b100de0943e09ce2b82ce611055dbdd095f120/integration/environment/machine.go#L31-L32

Reply:
yeah, thanks!